elixirkit_next/examples/cli_example/src-elixir/lib/example.ex

```diff
@@ -0,0 +1,43 @@
+defmodule Example.Application do
+  @moduledoc false
+  use Application
+
+  @impl true
+  def start(_type, _args) do
+    children = [Example.Server]
+    opts = [strategy: :one_for_one, name: Example.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+end
+
+defmodule Example.Server do
+  @moduledoc false
+  use GenServer
+
+  def start_link(arg) do
+    GenServer.start_link(__MODULE__, arg, name: __MODULE__)
+  end
+
+  @impl true
+  def init(_) do
+    {:ok, pid} = ElixirKit.start()
```

Comment:
We probably want to link those two processes anyway... also, what happens if ElixirKit is called multiple times?

Another option is for you to automatically start the ElixirKit process when ElixirKit starts and implement a pubsub using Elixir's Registry...

Reply:
Agreed, I'll revisit this soon. I even wanna rename ElixirKit.publish to ElixirKit.send exactly because I don't think we wanna build pubsub (one could fan out off ElixirKit listener process I suppose though)